docs/content/direct/wec-registration.md

```diff
@@ -1,5 +1,307 @@
 # Registering a Workload Execution Cluster
 
-This document will tell users what how to register a WEC in an ITS.
+This document explains how to register a Workload Execution Cluster (WEC) with an Inventory and Transport Space (ITS) in KubeStellar.
 
-TODO: write this.
+## Overview
+
+Registering a WEC in an ITS is the same process as registering a managed cluster with an OCM hub cluster. KubeStellar uses Open Cluster Management (OCM) for cluster registration and management.
+
+
+### Terminology Mapping
+
+| OCM Term             | KubeStellar Equivalent                        |
+|----------------------|-----------------------------------------------|
+| **OCM Hub**          | **KubeStellar ITS** (Inventory and Transport Space) |
+| **OCM Managed Cluster** | **KubeStellar WEC** (Workload Execution Cluster) |
+| **OCM Agent**        | **OCM Agent** (same component, installed on WEC) |
+
+For the complete OCM registration process, refer to the [official Open Cluster Management documentation](https://open-cluster-management.io/docs/getting-started/installation/register-a-cluster/).
+
+## Prerequisites
+
+Before registering a WEC, ensure you have:
+
+1. **A running Kubernetes cluster** that will serve as the WEC
+2. **Network connectivity** between the WEC and the ITS
```

Comment:
I think that it is true and worthwhile to say that what is required is that HTTPS connections can be opened from the WEC to the ITS.